--- Makefile~	2002-10-31 18:47:10.000000000 +0100
+++ Makefile	2003-01-25 23:40:58.000000000 +0100
@@ -6,7 +6,7 @@
 CC=		gcc
 GCCFLAGS=	-Wall -ansi -pedantic
 # HP-UX users may need to add -D_HPUX_SOURCE
-CFLAGS=		$(GCCFLAGS) $(DEBUG) -O -DCHAR_SIZE=$(CHAR_SIZE)
+CFLAGS=		$(RPM_OPT_FLAGS) -DCHAR_SIZE=$(CHAR_SIZE)
 LDFLAGS=	$(DEBUG)
 
 # You may well have to change this line.  Just "LIBS=" will probably work.
